Tests/js/views/fields/assets/ez-author-view-tests.js

```diff
@@ -20,6 +20,21 @@ YUI.add('ez-author-view-tests', function (Y) {
                 });
             },
 
+            "Test isEmpty with an empty field": function () {
+                this._testIsEmpty(
+                    {}, true,
+                    "The field should be seen as empty"
+                );
+            },
+
+            "Test isEmpty with a filled fieldValue": function () {
+                this._testIsEmpty(
+                    this.field,
+                    false,
+                    "The field should not be seen as empty"
+                );
+            },
+
```

Comment:
To completely cover the added code, you should also add a test where the field contains `{fieldValue: []}`